excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/DisplayGroupParserTest.java

```diff
@@ -418,14 +422,15 @@ void shouldFailForInvalidCaseRole() throws InvalidShowConditionException {
         item1.addAttribute(ColumnName.CASE_TYPE_ID.toString(), CASE_TYPE_UNDER_TEST);
         item1.addAttribute(ColumnName.TAB_ID.toString(), "Name Tab");
         item1.addAttribute(ColumnName.TAB_LABEL.toString(), "Name");
-        item1.addAttribute(ColumnName.USER_ROLE.toString(), caseRole);
+        item1.addAttribute(ColumnName.ACCESS_PROFILE.toString(), caseRole);
         item1.addAttribute(ColumnName.TAB_DISPLAY_ORDER.toString(), 1.0);
         item1.addAttribute(ColumnName.CASE_FIELD_ID.toString(), "PersonFirstName");
         definitionSheet.addDataItem(item1);
         MapperException thrown = assertThrows(MapperException.class,
             () -> caseTypeTabParser.parseAll(definitionSheets));
         assertThat(thrown.getMessage(),
-            is("- Invalid idam or case role '[CLAIMANT]' in 'CaseTypeTab' tab for TabId 'Name Tab'"));
+            is("- Invalid IDAM access profile or case role '[CLAIMANT]' in 'CaseTypeTab' tab "
```

Comment:
Rename to remove reference to IDAM.

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f